### ISSUE TITLE
Add a section for deploying and interacting with contracts via Contracts UI

### ISSUE THREAD

**Issue by ical10** [r0gue-io/pop-docs#8]
![Screenshot 2024-08-16 at 19 18 29](https://github.com/user-attachments/assets/97f53cda-ea29-428e-9239-ada52b13e3d3)

Since Pop Network testnet is now supported by Contracts UI, it would be a nice addition to have a small section explaining how to deploy and interact with contracts via this UI.

**Comment by brunopgalvao** [comment]
Yes good feedback, we also recently deployed: https://contracts.onpop.io
